tools/monitoring/Makefile

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all:
```

Comment:
do we need this file? I can do `go build .` directly?

Reply:
You can also do `go build .` directly for now. This make file is used in building the container. Once we start adding dependencies (google cloud apis), we can add `go get ..` directly in this file to manage the container go dependencies. 

```
# From images/monitoring/Dockerfile, where it uses this Makefile
RUN make -C $TEMP_TOOLS_REPO_DIR/$TOOL_NAME/
RUN cp $TEMP_TOOLS_REPO_DIR/$TOOL_NAME/$TOOL_NAME /$TOOL_NAME
```

Other code sample in test-infra using this pattern: https://github.com/knative/test-infra/blob/master/tools/flaky-test-reporter/Makefile

Reply:
sgtm